<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>974</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>974</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Заполнение областей</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="maximumSize">
      <size>
       <width>330</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>310</width>
          <height>670</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>670</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">* {
font: 14pt &quot;Ubuntu&quot;;
}

QFrame {
border: 0;
padding: 0;
margin: 0;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>G:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>R:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2" colspan="5">
          <widget class="QSpinBox" name="r">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Z кон.:</string>
           </property>
          </widget>
         </item>
         <item row="17" column="2">
          <widget class="Line" name="line_5">
           <property name="styleSheet">
            <string notr="true">border: 1px solid lightgray;
background-color: rgb(211, 215, 207);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="17" column="6">
          <widget class="QSpinBox" name="dx">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="18" column="1">
          <widget class="QSpinBox" name="x_end">
           <property name="minimum">
            <number>-10000</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>350</number>
           </property>
          </widget>
         </item>
         <item row="17" column="1">
          <widget class="QSpinBox" name="x0">
           <property name="minimum">
            <number>-10000</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>-350</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>B:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="7">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Отображаемая поверхность:</string>
           </property>
          </widget>
         </item>
         <item row="14" column="5" colspan="2">
          <widget class="QSpinBox" name="dz">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="5" column="2" colspan="5">
          <widget class="QSpinBox" name="g">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>2000</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="16" column="0" colspan="7">
          <widget class="Line" name="line_9">
           <property name="styleSheet">
            <string notr="true">border: 1px solid lightgray;
background-color: rgb(211, 215, 207);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="7">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Углы поворота поверхности:</string>
           </property>
          </widget>
         </item>
         <item row="17" column="3" colspan="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>dX:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="7">
          <widget class="Line" name="line_3">
           <property name="styleSheet">
            <string notr="true">border: 1px solid lightgray;
background-color: rgb(211, 215, 207);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="7">
          <widget class="Line" name="line_6">
           <property name="styleSheet">
            <string notr="true">border: 1px solid lightgray;
background-color: rgb(211, 215, 207);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QSpinBox" name="z0">
           <property name="minimum">
            <number>-10000</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>-150</number>
           </property>
          </widget>
         </item>
         <item row="18" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>X кон.:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2" colspan="5">
          <widget class="QSpinBox" name="b">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Цвет линий:</string>
           </property>
          </widget>
         </item>
         <item row="15" column="2">
          <widget class="Line" name="line">
           <property name="styleSheet">
            <string notr="true">border: 1px solid lightgray;
background-color: rgb(211, 215, 207);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="13" column="0" colspan="7">
          <widget class="Line" name="line_8">
           <property name="styleSheet">
            <string notr="true">border: 1px solid lightgray;
background-color: rgb(211, 215, 207);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Z нач.:</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QSpinBox" name="z_end">
           <property name="minimum">
            <number>-10000</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>150</number>
           </property>
          </widget>
         </item>
         <item row="17" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>X нач.:</string>
           </property>
          </widget>
         </item>
         <item row="19" column="0" colspan="7">
          <widget class="Line" name="line_10">
           <property name="styleSheet">
            <string notr="true">border: 1px solid lightgray;
background-color: rgb(211, 215, 207);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="14" column="2">
          <widget class="Line" name="line_4">
           <property name="styleSheet">
            <string notr="true">border: 1px solid lightgray;
background-color: rgb(211, 215, 207);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="14" column="3" colspan="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>dZ:</string>
           </property>
          </widget>
         </item>
         <item row="18" column="2">
          <widget class="Line" name="line_2">
           <property name="styleSheet">
            <string notr="true">border: 1px solid lightgray;
background-color: rgb(211, 215, 207);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QSpinBox" name="x_angle">
           <property name="minimum">
            <number>-360</number>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QSpinBox" name="y_angle">
           <property name="minimum">
            <number>-360</number>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Z:</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QSpinBox" name="z_angle">
           <property name="minimum">
            <number>-10000</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="7">
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item row="20" column="0" colspan="7">
          <widget class="QPushButton" name="addpoint_6">
           <property name="text">
            <string>Построить</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addpoint_6</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>render()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1039</x>
     <y>663</y>
    </hint>
    <hint type="destinationlabel">
     <x>1213</x>
     <y>665</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>rotateOx()</slot>
  <slot>rotateOy()</slot>
  <slot>rotateOz()</slot>
  <slot>render()</slot>
 </slots>
</ui>
